src/components/PageSwitch.tsx

```diff
@@ -102,6 +106,7 @@ const PageSwitch = ({
       {isSettingPageManually ? (
         <PageManualInput
           step={1}
+          type="number"
```

Comment:
What do you think about this? It should probably trigger the number pad on mobile. It also renders the up/down arrows on the browser.

Reply:
I started with adding the type, then removed it and forgot to remove the "step" as well.

When specifying `number` as a type, I couldn't prevent the input from showing letters (which doesn't make any sense IMO. Why allow a "number" input to display anything else than numbers..).

Reply:
ok, feel free to remove it, I'll focus on the windows thing today again...